extensions/ccpool/node-pool.ts

```diff
@@ -162,7 +162,7 @@ export class NodePool {
             // @ts-ignore
             const handler = this.poolHandlerComp ? obj.getComponent(this.poolHandlerComp) : null;
             if (handler && handler.reuse) {
-                handler.reuse(arguments);
+                handler.reuse(...args);
```

Comment:
There is a lint error say: `A spread argument must either have a tuple type or be passed to a rest parameter.`

I think here should be `handler.reuse(args);`